### PR TITLE
Add tags to cloud build from target_experiment.py

### PR DESCRIPTION
**Pull request by DonggeLiu**
This helps us to identify and debug cloud build from OFG.

### CODE REVIEW

**Comment by DonggeLiu** [comment]
This adds tag `dg` and `test` as expected to this test cloud build:
https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/667cb8e8-5693-4299-9666-c1da514b8615?e=-13802955&mods=logs_tg_prod&project=oss-fuzz

File: infra/build/functions/target_experiment.py [hunk 1]
```diff
@@ -279,15 +279,13 @@ def run_experiment(project_name, target_name, args, output_path, errlog_path,
...
   # Empirically, 3 hours is more than enough for 30-minute fuzzing cloud builds.
   build_lib.BUILD_TIMEOUT = 3 * 60 * 60
-  build_id = build_project.run_build(project_name,
-                                     steps,
-                                     credentials,
-                                     'experiment',
-                                     experiment=True,
-                                     extra_tags=[
-                                         f'experiment-{experiment_name}',
-                                         f'experiment-{project_name}'
```
Comment (DonggeLiu) [inline infra/build/functions/target_experiment.py:289]: Removed the `experiment-` prefix to be consistent with tags from OFG.
Please let me know if keeping the prefix is preferred, I will fix OFG.

**Review by oliverchang — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
I think this may have introduced a regression?

```sh
2024-09-20 22:39:56.888 INFO models - do_generate: vertex_ai_gemini-1-5 generating response with config: {'temperature': 0.6, 'max_output_tokens': 8192}
2024-09-20 22:39:56.906 INFO models - do_generate: vertex_ai_gemini-1-5 generating response with config: {'temperature': 0.6, 'max_output_tokens': 8192}
2024-09-20 22:39:57.088 ERROR builder_runner - _run_with_retry_control: Failed to evaluate /experiment/results/output-htslib-hfile_list_plugins/fixed_targets/02.c on cloud, attempt 1:

Traceback (most recent call last):	  File "/tmp/tmp5dlnpp7z/infra/build/functions/target_experiment.py", line 339, in <module>	    main()	  File "/tmp/tmp5dlnpp7z/infra/build/functions/target_experiment.py", line 332, in main	    run_experiment(args.project, args.target, args.args, args.upload_output_log,	  File "/tmp/tmp5dlnpp7z/infra/build/functions/target_experiment.py", line 288, in run_experiment	    extra_tags=[experiment_name, project_name] + tags)	               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~	TypeError: can only concatenate list (not "NoneType") to list	
2024-09-20 22:39:57.088 INFO evaluator - log: Fixing /experiment/results/output-htslib-hfile_list_plugins/fixed_targets/02.c with vertex_ai_gemini-1-5, attempt 1.
2024-09-20 22:39:57.088 WARNING prompt_builder - _format_fixer_problem: Unexpected empty error message in fix prompt for error_desc: None
```

From https://github.com/google/oss-fuzz-gen/pull/637

**Comment by DonggeLiu** [comment]
> I think this may have introduced a regression?
> 
> ```shell
> 2024-09-20 22:39:56.888 INFO models - do_generate: vertex_ai_gemini-1-5 generating response with config: {'temperature': 0.6, 'max_output_tokens': 8192}
> 2024-09-20 22:39:56.906 INFO models - do_generate: vertex_ai_gemini-1-5 generating response with config: {'temperature': 0.6, 'max_output_tokens': 8192}
> 2024-09-20 22:39:57.088 ERROR builder_runner - _run_with_retry_control: Failed to evaluate /experiment/results/output-htslib-hfile_list_plugins/fixed_targets/02.c on cloud, attempt 1:
> 
> Traceback (most recent call last):	  File "/tmp/tmp5dlnpp7z/infra/build/functions/target_experiment.py", line 339, in <module>	    main()	  File "/tmp/tmp5dlnpp7z/infra/build/functions/target_experiment.py", line 332, in main	    run_experiment(args.project, args.target, args.args, args.upload_output_log,	  File "/tmp/tmp5dlnpp7z/infra/build/functions/target_experiment.py", line 288, in run_experiment	    extra_tags=[experiment_name, project_name] + tags)	               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~	TypeError: can only concatenate list (not "NoneType") to list	
> 2024-09-20 22:39:57.088 INFO evaluator - log: Fixing /experiment/results/output-htslib-hfile_list_plugins/fixed_targets/02.c with vertex_ai_gemini-1-5, attempt 1.
> 2024-09-20 22:39:57.088 WARNING prompt_builder - _format_fixer_problem: Unexpected empty error message in fix prompt for error_desc: None
> ```
> 
> From [google/oss-fuzz-gen#637](https://github.com/google/oss-fuzz-gen/pull/637)

Thanks! I will fix this today.

**Pull request merged**